### PR TITLE
[CI][main] Update release workflows, add update header workflow

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] PR merge commit message adjusted

### CODE REVIEW

File: changelog.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+commit_types = [
+    { message = "^add", group = "Added"},
+    { message = "^remove", group = "Removed"},
+    { message = "^change", group = "Changed"},
+    { message = "^fix", group = "Bug Fixes"},
+]
```
Comment (ArnoStiefvater) [inline changelog.toml:6]: In gvm-libs there is also 
```    
    { message = "^doc", group = "Documentation"},
    { message = "^refactor", group = "Refactor"},
    { message = "^test", group = "Testing"},
```

**Review by ArnoStiefvater — APPROVED** [review]

File: changelog.toml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+commit_types = [
+    { message = "^add", group = "Added"},
+    { message = "^remove", group = "Removed"},
+    { message = "^change", group = "Changed"},
+    { message = "^fix", group = "Bug Fixes"},
```
Comment (Dexus) [inline changelog.toml:5]:
```suggestion
    { message = "^add", group = "Added"},
    { message = "^remove", group = "Removed"},
    { message = "^change", group = "Changed"},
    { message = "^fix", group = "Bug Fixes"},
    { message = "^doc", group = "Documentation"},
    { message = "^refactor", group = "Refactor"},
    { message = "^test", group = "Testing"},
```

**Review by Dexus — CHANGES_REQUESTED** [review]

**Review by Dexus — APPROVED** [review]

File: .github/workflows/release-pontos-manually.yml [hunk 3]
```diff
@@ -5,50 +5,18 @@ on:
...
-      GITHUB_TOKEN: ${{ secrets.GREENBONE_BOT_TOKEN }}
-      GPG_KEY: ${{ secrets.GPG_KEY }}
-      GPG_FINGERPRINT: ${{ secrets.GPG_FINGERPRINT }}
-      GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
     name: Release patch with pontos
     runs-on: 'ubuntu-latest'
     steps:
-    - uses: actions/checkout@v2
+    - name: Release with release action
+      uses: greenbone/actions/release-python@v1
```
Comment (Dexus) [inline .github/workflows/release-pontos-manually.yml:12]: please review https://github.com/greenbone/actions/pull/38 before merge!

**Review by Dexus — CHANGES_REQUESTED** [review]

File: changelog.toml (hunk 2 above)
Comment (y0urself) [reply changelog.toml:5]: We do not want test, refactor, and doc in the release data, thus no. :D

**Pull request merged**